src/redux/data.ts

```diff
@@ -107,15 +131,9 @@ export const COINGECKO_IDS_ENDPOINT =
 
 // -- Constants --------------------------------------- //
 
-const DATA_UPDATE_ASSET_PRICES_FROM_UNISWAP =
-  'data/DATA_UPDATE_ASSET_PRICES_FROM_UNISWAP';
-const DATA_UPDATE_ACCOUNT_ASSETS_DATA = 'data/DATA_UPDATE_ACCOUNT_ASSETS_DATA';
-
 const DATA_UPDATE_GENERIC_ASSETS = 'data/DATA_UPDATE_GENERIC_ASSETS';
 const DATA_UPDATE_ETH_USD = 'data/DATA_UPDATE_ETH_USD';
-const DATA_UPDATE_ETH_USD_CHARTS = 'data/DATA_UPDATE_ETH_USD_CHARTS';
 const DATA_UPDATE_PORTFOLIOS = 'data/DATA_UPDATE_PORTFOLIOS';
-const DATA_UPDATE_TRANSACTIONS = 'data/DATA_UPDATE_TRANSACTIONS';
 const DATA_UPDATE_UNISWAP_PRICES_SUBSCRIPTION =
   'data/DATA_UPDATE_UNISWAP_PRICES_SUBSCRIPTION';
```

Comment:
There were some actions that had the same effect as others (such as `DATA_UPDATE_ASSET_PRICES_FROM_UNISWAP` and `DATA_LOAD_ASSET_PRICES_FROM_UNISWAP_SUCCESS`) so I changed all instances over to a single action.